### PR TITLE
added product listing

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (829)
Screenshot (830)

### CODE REVIEW

**Review by iHarryD — APPROVED** [review]
Everything looks good but would recommend using uppercase letters for variable names.

**Comment by ashishkatiyar2404** [comment]
> Everything looks good but would recommend using uppercase letters for variable names.

ok will try after submision

File: library/Components/product/product.html [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+          <div>
+            <input type="radio" class="two" name="agree" />
+            <label for="two">2 stars & above</label>
+          </div>
+        </div>
+        <div class="product__sorting">
+          <span>Sort By</span>
+          <div>
+            <input type="radio" class="low" name="agree" />
+            <label for="three">Price - low to high</label>
```
Comment (pradipgharge) [inline library/Components/product/product.html:113]: input should have an id which should be given to for attribute of the label

**Review by pradipgharge — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Hello, beautiful work on the UI there, it's really nice that you have provided the screenshot for it, now coming to your code , your code is also good according to me, I understood the class names as soon as I saw them, and maybe you should look into consistent indentations a bit, because in some places I saw no spacing and in some there was. But really good work :)

File: library/Components/product/product.html (hunk 1 above)
Comment (ashishkatiyar2404) [reply library/Components/product/product.html:113]: ok. That I was missed. thanks

**Pull request merged**